cli/src/main.rs

```diff
@@ -35,21 +35,55 @@ mod args;
 async fn main() {
     let args = Args::parse();
     match args.command {
-        Command::Start(start_args) => {
-            if start_args.verbose {
+        Command::Start(start) => {
+            if start.verbose {
                 tracing_subscriber::fmt()
                     .with_max_level(tracing::Level::DEBUG)
                     .init()
             } else {
                 tracing_subscriber::fmt::init();
             }
-            info!("Starting server on port {}", start_args.port);
-            let server = Server::new(start_args.port);
+            info!("Starting server on port {}", start.port);
+            if start.detach {
+                match std::env::current_exe() {
+                    Ok(buf) => {
+                        let mut command = std::process::Command::new(buf);
+                        command
+                            .arg("start")
+                            .arg("--port")
+                            .arg(start.port.to_string())
+                            .stdout(std::process::Stdio::null())
+                            .stderr(std::process::Stdio::null())
+                            .stdin(std::process::Stdio::null());
+                        if start.verbose {
+                            command.arg("--verbose");
+                        }
+                        #[cfg(target_os = "windows")]
+                        use std::os::windows::process::CommandExt;
+                        #[cfg(target_os = "windows")]
+                        command.creation_flags(0x00000008);
```

Comment:
@h1alexbel `0x00000008` – some magic value

Reply:
@l3r8yJ actually constant here causes more trouble - it will require `u32` instead of `i32` on windows

Reply:
@h1alexbel in this case we can ad a comment